Shared/Extensions/URLExtensions.swift

```diff
@@ -440,6 +440,22 @@ extension URL {
         
         return siteList.contains(where: domain.contains)
     }
+    
+    // Check if the website is supporting showing Add To playlist toast
+    public var isPlaylistSupportedSiteURL: Bool {
+        guard let domain = self.baseDomain else {
+            return false
```

Comment:
this will return true for sites like `youtube.evil.com`, I think `self.hostSLD` will work better.